.circleci/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM circleci/clojure:lein-2.7.1
+LABEL maintainer="hlship@gmail.com"
+
+# Based on notes here:
+# https://www.metachris.com/2017/01/how-to-install-nodejs-7-on-ubuntu-and-centos/
+
+RUN curl -sL https://deb.nodesource.com/setup_7.x | sudo bash -
+RUN sudo apt-get install -y nodejs
+
+# docker build -t hlship/circleci-lein-node:2.7.1 .
```

Comment:
Uh, subscriptions.clj is 404 lines with comments.  I think having multiple go loops keeps things simpler, connection-loop would be even larger if we folded the other things in.  But note this is *initial* support.  

It was only after I dug a bit that I found out I wouldn't be able to re-use the interceptors, but create new parallel ones. Maybe that's for the best.

Thanks for the review!